charts/erigon/README.md

```diff
@@ -56,59 +56,59 @@ We do not recommend that you upgrade the application by overriding `image.tag`.
 
 ## Values
 
-| Key | Type | Default | Description |
```

Comment:
Excuse this diff in here. I edited a global template, hence it re-rendering for other charts too.